src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-form/iscsi-target-form.component.ts

```diff
@@ -74,6 +75,8 @@ export class IscsiTargetFormComponent implements OnInit {
   IQN_REGEX = /^iqn\.(19|20)\d\d-(0[1-9]|1[0-2])\.\D{2,3}(\.[A-Za-z0-9-]+)+(:[A-Za-z0-9-\.]+)*$/;
   USER_REGEX = /[\w\.:@_-]{8,64}/;
   PASSWORD_REGEX = /[\w@\-_\/]{12,16}/;
+  action: string;
+  resource = 'target';
```

Comment:
This needs to be translated.
Either translate it in the TS or remove the variable and translate in the HTML.

Same applies to all other `resource` vars.

Reply:
I initially assumed that probably most 'resources' would go untranslated, as they were very so specific that would remain in 'English' (Pool, bucket, RBD image), but others are easier (e.g.: user). So I'll do that.